src/ForumFiltering.js

```diff
@@ -1,16 +1,18 @@
 import React from "react";
 import "./ForumFiltering.css";
+import axios from 'axios';
 
-function ForumFiltering (){
-    var keyword1 = {word: "Sports", type: "keyWords", checked: false}
-    var keyword2 = {word: "Community", type: "keyWords", checked: false}
-    var keyword3 = {word: "School", type: "keyWords", checked: false}
-    var keyword4 = {word: "Music", type: "keyWords", checked: false}
-    var keyword5 = {word: "Art", type: "keyWords", checked: false}
-    var cost1 = {word: "Free", type: "Cost", checked: false}
-    var cost2 = {word: "About 5 Dollars", type: "Cost", checked: false}
-    var cost3 = {word: "About 10 Dollars", type: "Cost", checked: false}
-    var cost4 = {word: "More than 10 Dollars", type: "Cost", checked: false}
+
+function ForumFiltering (props){
```

Comment:
We might try and get all the filters on one line, like this:

Sports | Community | School | Music | Art
    O              O                O          O       O

If not, this works just fine.

Reply:
Haha the formatting of the comment screwed up, but I think it's clear.